cli/onionshare_cli/resources/static/js/socket.io.min.js

```diff
@@ -1,7 +1,7 @@
 /*!
- * Socket.IO v3.1.1
- * (c) 2014-2021 Guillermo Rauch
+ * Socket.IO v3.0.4
```

Comment:
This downgrade is not on purpose and due to being behind upstream I assume?

socket.io was upgraded in #1285

Reply:
Thanks for noticing. That was not on purpose. Let me update it to the latest version and see if it works.